src/w2grid.js

```diff
@@ -387,6 +387,20 @@
 
         get: function (recid, returnIndex) {
             // search records
+            if ($.isArray(recid)) {
+                var returnrecs = [];
+                for (var i = 0; i < this.records.length; i++) {
+                    if ($.inArray(this.records[i].recid, recid) != -1) {
+                        if (returnIndex === true) returnrecs.push({'store': 'records', 'index': i}); else returnrecs.push(this.records[i]); 
+                    }
+                }
+                for (var i = 0; i < this.summary.length; i++) {
+                    if ($.inArray(this.summary[i].recid, recid) != -1) {
+                        if (returnIndex === true) returnrecs.push({'store': 'summary', 'index': i}); else returnrecs.push(this.summary[i]);
+                    }
+                }
+                return $.isEmptyObject(returnrecs) ? null : returnrecs;
```

Comment:
Also, I think it should return [] if no records found. For example, you do
```js
var recs = grid.get([1,2,3]);
if (recs.length != 3) alert('error');
```
Otherwise, you will need 2 conditions, (1) check for null, (2) check if my records is returns.

I will do the changes.